test.js

```diff
@@ -47,6 +53,12 @@ test('Get ID by English name', testIDs, 'en', [
 	'Volcanion'
 ]);
 
+test('Get ID by French name', testIDs, 'fr', [
```

Comment:
We don't need this test for every new language. It's enough to test the `Get French name by ID`. Everything else is already tested by English and German.